src/garage/experiment/experiment.py

```diff
@@ -328,14 +289,17 @@ def run_experiment(method_call=None,
             data = base64.b64encode(pickle.dumps(call)).decode('utf-8')
         task['args_data'] = data
         exp_count += 1
-        params = dict(kwargs)
+
         if task.get('exp_name', None) is None:
             task['exp_name'] = '%s_%s_%04d' % (exp_prefix, timestamp,
                                                exp_count)
         if task.get('log_dir', None) is None:
-            task['log_dir'] = (
-                config.GARAGE_LOG_DIR + '/local/' + exp_prefix.replace(
-                    '_', '-') + '/' + task['exp_name'])
+            garage_project_path = osp.abspath(
```

Comment:
this is not a great way to do this. it is doing to put data inside a git repository (or possibly installation directory, if garage is installed using pip). 

I think the default should be `'data'`, in the same directory as the launcher script.